src/server/api/routes/api-router.js

```diff
@@ -4,7 +4,13 @@ const router = express.Router();
 
 // Router imports
 const modulesRouter = require('./modules.router');
+<<<<<<< HEAD
+
+const usersRouter = require('./users.router');
+
+=======
 const usersRouter = require('./users-router');
+>>>>>>> develop
```

Comment:
**Never ever** commit a merge conflict. This is absolutely a no go!

Watch this: https://www.youtube.com/watch?v=JtIX3HJKwfo

Remember your code does not run now. So it shows that you have not even bothered to run your code before making the pr.